shared/src/commonMain/graphql/schema.graphqls

```diff
@@ -62,27 +62,27 @@ type Session {
 
   description: String
 
-  endInstant: Instant!
+  endInstant: Instant! @requiresOptIn(feature: "experimental")
 
   feedbackId: String
 
   id: String!
 
   language: String
 
-  startInstant: Instant!
+  room: Room
 
-  tags: [String!]!
+  rooms: [Room!]!
 
-  title: String!
+  speakers: [Speaker!]!
 
-  type: String!
+  startInstant: Instant! @requiresOptIn(feature: "experimental")
 
-  room: Room @deprecated(reason: "use rooms instead")
```

Comment:
This `@deprecated` shouldn't have been removed. See https://github.com/joreilly/Confetti/pull/185/files